private/projects/pdk-monorepo-project.ts

```diff
@@ -175,9 +172,15 @@ export class PDKMonorepoProject extends NxMonorepoProject {
       ".pnp.loader.cjs"
     );
 
+    const npmrc = new IniFile(this, ".npmrc", {});
     // add local `.npmrc` to automatically avoid build hangs if npx is promping to install a package
-    const npmrc = new NpmConfig(this);
-    npmrc.addConfig("yes", "true");
+    npmrc.addOverride("yes", true);
+    // https://pnpm.io/npmrc#public-hoist-pattern
+    npmrc.addOverride("public-hoist-pattern", [
```

Comment:
Why do we need to hoist these?

Reply:
eslint and prettier are the defaults for pnpm `public-hoist-pattern`, so need append `@pnpm/*` also needed to add these back. The `@pnpm/*` was needed when I used the pnpm lock instead of pnpm list to get deps and path - but not needed now.

I will remove this chunk - not needed now.